.github/workflows/lint.yml

```diff
@@ -24,10 +24,11 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.7
+        python-version: 3.9
 
+    # flake8 6.0.0 removed the --diff option
     - name: Install dependencies
-      run: pip install -U flake8 wheel setuptools
+      run: pip install --upgrade wheel setuptools "flake8<6.0.0"
```

Comment:
I'll just fix the existing flake8 issues. If I run `autopep8` on the codebase, that reduces it down to a manageable amount.

Reply:
OK, that sounds like a good plan. Perhaps we could enforce `yapf` formatting on this repository as well?

Reply:
Do you want yapf? I've rather lost my liking for it and it's stubborn habit of rewrapping things in ways I didn't want.

Reply:
I've gotten used to it in the main repository, so I wouldn't mind having it here, either.